bundle/artifacts/artifacts.go

```diff
@@ -186,6 +186,9 @@ func rewriteArtifactPath(b *bundle.Bundle, f *config.ArtifactFile, job *resource
 			if lib.Jar != "" && isArtifactMatchLibrary(f, lib.Jar, b) {
 				lib.Jar = remotePath
 			}
+			if lib.Requirements != "" {
```

Comment:
Now with having translate_path change for requirements I don't think you need this